public/docs/ts/latest/guide/attribute-directives.jade

```diff
@@ -89,167 +92,165 @@ include ../_quickstart_repo
+  +ifDocsFor('ts')
+    | We export `HighlightDirective` to make it accessible to other components.
 :marked
-  After the `@Directive` metadata comes the directive's controller class which we are exporting
-  to make it accessible to other components. 
-  The directive's controller class contains the logic for the directive.
-  
   Angular creates a new instance of the directive's controller class for
   each matching element, injecting an Angular `ElementRef` 
   into the constructor.
-  
   `ElementRef` is a service that grants us direct access to the DOM element
   through its `nativeElement` property.
   That's all we need to set the element's background color using the browser DOM API.
 
 .l-main-section
+a#apply-directive
 :marked
   ## Apply the attribute directive
   The `AppComponent` in this sample is a test harness for our `HighlightDirective`.
   Let's give it a new template that 
-  applies the directive as an attribute to a `span` element.
-  In Angular terms, the `<span>` element will be the attribute **host**.
-  
-  We'll put the template in its own `app.component.html` file that looks like this:
+  applies the directive as an attribute to a paragraph (`p`) element.
+  In Angular terms, the `<p>` element will be the attribute **host**.
+p
+  | We'll put the template in its own 
+  code #[+adjExPath('app.component.html')]
+  | file that looks like this:
 +makeExample('attribute-directives/ts/app/app.component.1.html',null,'app/app.component.html')(format=".")
 :marked
   A separate template file is clearly overkill for a 2-line template. 
   Hang in there; we're going to expand it later.
   Meanwhile, we'll revise the `AppComponent` to reference this template.
 +makeExample('attribute-directives/ts/app/app.component.ts',null,'app/app.component.ts')
 :marked
-  We've added an `import` statement to fetch the 'Highlight' directive and 
-  added that class to a `directives` array in the component metadata so that Angular 
+  We've added an `import` statement to fetch the 'Highlight' directive and, 
+  added that class to a `directives` component metadata so that Angular 
   will recognize our directive when it encounters `myHighlight` in the template. 
 
-  We run the app and see that our directive highlights the span text.
+  We run the app and see that our directive highlights the paragraph text.
   
 figure.image-display
   img(src="/resources/images/devguide/attribute-directives/first-highlight.png" alt="First Highlight")
 .l-sub-section
   :marked
-    #### Why isn't my directive working? 
+    ### Your directive isn't working? 
 
-    Did you remember to set the `directives` array? It is easy to forget!
+    Did you remember to set the `directives` attribute of `@Component`? It is easy to forget!
     
     Open the console in the browser tools and look for an error like this:
-  code-example.format("").
+  code-example(format="nocode").
```

Comment:
It tells the prettifier not to apply any syntax highlighting (previously the error message words were in different colors).